responsemanager/queryexecutor.go

```diff
@@ -72,6 +72,7 @@ func (qe *queryExecutor) processQueriesWorker() {
 				log.Info("Empty task on peer request stack")
 				continue
 			}
+			log.Debugf("Beginning response execution: ID: %d, Peer: %s, Root CID: %s", key.requestID, key.p.String(), taskData.request.Root().String())
```

Comment:
Let's use structured logs.

```suggestion
			log.Debugw("beginning response execution", "id", key.requestID, "peer", key.p.String(), "root_cid", taskData.request.Root().String())
```